test-result-summary-client/src/PerfCompare/lib/JenkinsRunJSON.js

```diff
@@ -26,57 +23,23 @@ export default class JenkinsRunJSON {
                 // new variant
                 if (parsedVariantsCommon[curVariantObjectId] === undefined) {
                     parsedVariantsCommon[curVariantObjectId] = {
-                        product: (this.runJSON.testInfo[0].tests[k].benchmarkProduct === undefined) ? null : this.runJSON.testInfo[0].tests[k].benchmarkProduct,
-                        result: (this.runJSON.testInfo[0].tests[k].testResult === undefined) ? null : this.runJSON.testInfo[0].tests[k].testResult,
-                        benchmark: (this.runJSON.testInfo[0].tests[k].benchmarkName === undefined) ? null : this.runJSON.testInfo[0].tests[k].benchmarkName,
-                        variant: (this.runJSON.testInfo[0].tests[k].benchmarkVariant === undefined) ? null : this.runJSON.testInfo[0].tests[k].benchmarkVariant,
-                        machine: (this.runJSON.testInfo[0].machine === undefined) ? null : this.runJSON.testInfo[0].machine,
-                        metrics: (this.runJSON.testInfo[0].tests[k].testData.metrics === undefined) ? [] : this.runJSON.testInfo[0].tests[k].testData.metrics
+                        product: (aggregateInfo.benchmarkProduct === undefined) ? null : aggregateInfo.benchmarkProduct,
+                        result: (aggregateInfo.testResult === undefined) ? null : aggregateInfo.testResult,
```

Comment:
`aggregateInfo.testResult` seems like a bug since `aggregate` doesn't have that data.